main.go

```diff
@@ -230,9 +230,9 @@ func main() {
 		stdlog.Fatalf("parse flag: %v", err)
 	}
 
-	if !cfg.metrics.enabled && !cfg.logs.enabled {
+	if !cfg.metrics.enabled && !cfg.logs.enabled && !cfg.traces.enabled {
 		// Currently we don't support the case where only the gRPC trace endpoint is enabled
```

Comment:
This comment is not obsolete, right?

Reply:
I don't know, it's not the subject of this PR.

Reply:
actually yes, let me remove it